build: define typescript version via string in module.bazel file

josephperrott opened the pull request.

alan-agius4 reviewed (approved):
LGTM, might be worth considering moving this to dev-infra and re-use in all repos.

josephperrott committed:
build: define typescript version via string in module.bazel file

Within our module.bazel file when describing the version of typescript to use for rules_ts, we use ts_version instead of ts_version_from to prevent our package.json file from being
part of the set of files used to calculate the sha for the lock file.  Any unrelated change to the version of the typescript file would end up causing our lockfile to be out of date.
This amount of churn has proven to be too much for our current setup. We instead now test to validate the versions defined in the package.json and MODULE.bazel files match.

josephperrott commented:
@alan-agius4 PTAL as I moved to relying on the rule from dev-infra instead.

alan-agius4 reviewed (approved):
LGTM, thanks
